modules/core_project_factory/README.md

```diff
@@ -0,0 +1,74 @@
+# Core Project Factory
```

Comment:
These readme's are absent by design as these are considered internal modules. Let's remove any new readme and metadata added here and in the other two places.